Updated sqlcipher compile instructions for Fedora (Clarifies and closes  #2970)

linux-modder opened the pull request.

linux-modder committed:
Updated sqlcipher compile instructions for Fedora

zetok commented:
@suknight does that clarify things for you?

hethkar commented:
@zetok yes, thank you

zetok commented on INSTALL.md:
```diff
@@ -387,6 +388,24 @@ autoreconf -if
+$ ./configure --enable-tempstore=yes CFLAGS="-DSQLITE_HAS_CODEC" \
+    LDFLAGS="/opt/local/lib/libcrypto.a"
+$ make
+$ sudo make install
```
Could you remove all the unnecessary `$` ?

Also, put things that aren't commands outside of code block – each variant should be in a separate code block.

zetok commented on INSTALL.md:
````diff
@@ -380,6 +380,7 @@ sudo make install
 If you are not using Fedora, skip this section, and go directly to compiling [**toxcore**](#toxcore-compiling).
 
 ```bash
+This method Auto detects whether to statically or Dynamically  link per your system configs.
````
remove unneeded whitespace

zetok commented on INSTALL.md:
```diff
@@ -387,6 +388,24 @@ autoreconf -if
 cd ..
+
+
+Conversely, if you wish to explictly statically  or dynamically link sqlcipher use:
```
remove unneeded whitespace;

Why would `Conversely,` part be needed? It can start just from `If`

zetok commented on INSTALL.md:
`````diff
@@ -387,6 +388,24 @@ autoreconf -if
+$ make
+$ sudo make install
+cd ..
 ````
`````
while this isn't part of this changeset, it also could be changed to just    ```

zetok commented on INSTALL.md:
```diff
@@ -387,6 +388,24 @@ autoreconf -if
+$ sudo make install
+cd ..
+
+Dynamically Linked:
```
Can be `#### Dynamically linked`

zetok commented on INSTALL.md:
```diff
@@ -387,6 +388,24 @@ autoreconf -if
+
+Conversely, if you wish to explictly statically  or dynamically link sqlcipher use:
+
+Statically Linked:
```
Can be `#### Statically linked`

linux-modder committed:
Updated sqlcipher compile instructions for Fedora (Take #2)

- Cleaned up the code blocks
- Removed unneeded whitespace throughout previous commit

linux-modder committed:
Merge pull request #2 from linux-modder/linux-modder/sqlcipher

Updated sqlcipher compile instructions for Fedora (Take #2)

zetok commented:
Picked relevant commits in 6c0fb9a0f514d6b0ce82fe40db254f2cb31acfd6, thus closing.
